content/docs/user-guide/concepts/data-pipelines.md

```diff
@@ -0,0 +1,34 @@
+---
+name: 'Data Pipelines'
+match: ['data pipeline', 'pipeline', 'pipelines']
+tooltip: >-
+  A data pipeline is a series of data processing stages, chained by their
+  outputs and inputs. They use some initial data, may produce intermediate
+  artifacts, and reach a final result.
+  [📖](/doc/user-guide/concepts/data-pipelines)
+description: >-
+  A data pipeline is a series of data processing stages. They use some initial
+  data, may produce intermediate artifacts, and reach a final result.
+---
+
+<!-- keywords: data pipeline, machine learning pipeline, devops for data science, devops for machine learning, "MLops", "what is a data pipeline?", "data pipeline examples", "machine learning (ML) workflow", "data science workflow", "data science pipeline workflow" -->
+
```

Comment:
So these are desired terms we want to try to get into the concept text for SEO purposes.

Reply:
Writing a document that puts _all_ these words would be a challange 😄 

Instead there can be a `keywords:` front matter that will be copied to meta tags of pages. Talking to search engine bots and talking to humans are different purposes IMO. These keywords can also be added to other pages.

Reply:
Search engines don't really care about metadata keywords anymore, we need to incorporate the keywords to the content but yeah no need to have them all necessarily, especially in small concept pages that will be impossible.